### PR TITLE
drainer: fix too much version when initializing table infos

### PR DESCRIPTION
**Pull request by lichunzhu**
### What problem does this PR solve?

Issue Number: ref https://github.com/pingcap/tidb-binlog/issues/1137

### What is changed and how it works?
switch create table to batch create table to solve too many versions problem.

### Check List 

Tests 

 - Unit test
 - Integration test

Code changes

- None

Side effects

 - Increased code complexity

Related changes

- None

### Release note

- No release note

### CODE REVIEW

**Review by niubell — APPROVED** [review]
LGTM

File: drainer/config.go [hunk 1]
```diff
@@ -122,7 +122,7 @@ type SyncerConfig struct {
 	EnableCausalityFlag  *bool `toml:"-" json:"enable-detect-flag"`
 	DisableCausalityFile *bool `toml:"disable-detect" json:"disable-detect"`
 	EnableCausalityFile  *bool `toml:"enable-detect" json:"enable-detect"`
-	LoadTableInfos       bool  `toml:"load-table-infos" json:"load-table-infos"`
+	LoadSchemaSnapshot   bool  `toml:"load-schema-snapshot" json:"load-schema-snapshot"`
```
Comment (D3Hunter) [inline drainer/config.go:125]: in-compatible change? might need tell customer about this if they uses old parameter

**Review by D3Hunter — APPROVED** [review]

File: drainer/config.go (hunk 1 above)
Comment (lichunzhu) [reply drainer/config.go:125]: this parameter is never released. I have discussed with frank before and forget to change this.

**Pull request merged**